backend/spec/requests/memos_spec.rb

```diff
@@ -28,4 +28,48 @@
       end
     end
   end
+
+  describe 'PUT /memos/:id' do
+    context 'コンテンツが有効な場合' do
+      let(:existing_memo) { create(:memo) }
+      let(:update_memo_params) { { content: '新しいコンテンツ' } }
+
+      it 'memoが更新され、204になる' do
+        aggregate_failures do
+          put "/memos/#{existing_memo.id}", params: { memo: update_memo_params }
+          expect(response).to have_http_status(:no_content)
+          existing_memo.reload
+          expect(existing_memo.content).to eq('新しいコンテンツ')
+        end
+      end
+    end
+
+    context 'バリデーションエラーになる場合' do
+      let(:existing_memo) { create(:memo) }
+      let(:invalid_params) { { content: '' } }
```

Comment:
invalid_というprefixはなくても、contextを見ればinvalidなparamsを入れていることは想起できるかなと思いました〜
```suggestion
      let(:params) { { content: '' } }
```